igvm/vm.py

```diff
@@ -599,6 +601,8 @@ def build(
                 )
                 self.hypervisor.download_and_extract_image(image, mount_path)
 
+                self.hypervisor.release_lock()
```

Comment:
The hypervisor will never be unlocked if somebody is building a barebones VM.

Reply:
This function is executed inside a context manager, which will unlock the hypervisor always the latest that the function finshes